drivers/flash/soc_flash_nrf.c

```diff
@@ -77,6 +78,25 @@ static struct k_sem sem_lock;
 #define SYNC_UNLOCK()
 #endif
 
+#ifdef CONFIG_SOC_NRF9160_XXAA
```

Comment:
Shouldn't be SOC_NRF9160 or SOC_NRF9160_SICA
@anangl  ?

Reply:
Right. I overlooked this. It should be `CONFIG_SOC_NRF9160_SICA` (right now this is the only variant, but others may appear in some future). Otherwise, it won't work for non-secure builds.
Actually, it would be safer to use something like:
```
#if defined(CONFIG_SOC_SERIES_NRF51X) || defined(CONFIG_SOC_SERIES_NRF52X)
// use FICR
#elif defined(CONFIG_SOC_NRF9160_SICA)
// use hard-coded values
#else
// report an #error
#endif
```